experiment/cherrypicker/server.go

```diff
@@ -174,7 +174,7 @@ func (s *Server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 	if len(cherryPickMatches) == 0 || len(cherryPickMatches[0]) != 2 {
 		return nil
 	}
-	targetBranch := cherryPickMatches[0][1]
+	targetBranch := strings.TrimSpace(cherryPickMatches[0][1])
```

Comment:
Is it space that is breaking? I thought it was the fact that the last character in that title is an ellipsis and non-ascii?

Reply:
Special character (`\r` in this case) which is added by hiting Enter after the comment. See the unit test update. It's what we hit in the [first linked issue](https://github.com/openshift/origin/pull/15847#issuecomment-359410337) and I managed to reproduce in the [second](https://github.com/openshift/openshift-ansible/pull/6806#issuecomment-359500198).

Reply:
![image](https://user-images.githubusercontent.com/1537044/35286191-524616bc-002d-11e8-998c-b8a92d760448.png)

Reply:
OK, have you tested that the ellipsis also doesn't break things?